src/NextGen/WelcomeBanner/resources/views/welcome-banner.php

```diff
@@ -117,16 +132,16 @@
                             <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
                                 <path d="M10 4.166v11.667M4.165 9.999h11.667" stroke="#2D802F" stroke-width="1.667" stroke-linecap="round" stroke-linejoin="round"/>
                             </svg>
-                            <?php _e('Create a donation form using the new visual builder.', 'give'); ?>
+                            <?php _e('Create a next visual donation form', 'give'); ?>
```

Comment:
@JoshuaHungDinh This wording is weird. "Create a next visual"? I think it's intending to be "create a next gen visual donation form".

Reply:
Resolved: https://github.com/impress-org/givewp-next-gen/commit/3b0b52ca18864a1e23cfca25530e14a358dc1ae5